tests/ipex/test_modeling.py

```diff
@@ -220,6 +219,10 @@ def test_pipeline(self, model_arch):
         self.assertTrue(all("This is a sample" in item["generated_text"] for item in outputs))
 
     @parameterized.expand(SUPPORTED_ARCHITECTURES)
+    @unittest.skipIf(
+        is_ipex_version(">=", "2.3.0"),
+        reason="CPU IPEXModel does not support assisted decoding when ipex version >= 2.3.0",
+    )
```

Comment:
could you explain why assisted decoding is deprecated with v2.3.0?

Reply:
See [here](https://github.com/huggingface/optimum-intel/pull/725#issuecomment-2138517113)